portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,179 @@
-    <link rel="stylesheet" type="text/css" href="jasmine/lib/jasmine-3.5.0/jasmine.css" />
-    <script src="jasmine/lib/jasmine-3.5.0/jasmine.js"></script>
-    <script src="jasmine/lib/jasmine-3.5.0/jasmine-html.js"></script>
-    <script src="jasmine/lib/jasmine-3.5.0/boot.js"></script>
+    <script src="request.js"></script>
   </head>
-
   <body>
-    <div id="new-task-wrapper">
-      <h2>What are your working hours for today?</h2>
-      <span>
-        <input type="time" id="working-hour-start" name="working-hour-start" value="09:00" />
-        to
-        <input type="time" id="working-hour-end" name="working-hour-end" value="17:00" />
-      </span>
-
-      <h2>Add new tasks</h2>
-      <div id="new-task-form">
-        <label for="new-task-name">Task name:</label>
-        <input type="text" id="new-task-name" name="new-task-name" /><br />
-        <label for="new-task-description">Task description:</label><br />
-        <textarea id="new-task-description" name="new-task-description" cols="30" rows="10"></textarea><br />
-        <label for="new-task-estimated-length">Estimated length:</label>
-        <input type="text" id="new-task-estimated-length" name="new-task-estimated-length" />
-        <select id="new-task-estimated-length-unit" name="new-task-estimated-length-unit">
-          <option value="minute">minutes(s)</option>
-          <option value="hour">hour(s)</option>
-          <option value="day">day(s)</option>
-        </select>
-        <br />
-        <label for="new-task-priority">Task's priority:</label>
-        <select id="new-task-priority" name="new-task-priority">
-          <option value="5">5 (High)</option>
-          <option value="4">4</option>
-          <option value="3">3</option>
-          <option value="2">2</option>
-          <option value="1">1 (Low)</option> </select
-        ><br />
-        <button type="button" class="btn btn-primary">Add this task</button>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="#">Task Scheduler</a>
+      <div class="collapse navbar-collapse" id="navbar-supported-content">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              href="#"
+              id="navbar-dropdown"
+              role="button"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              Scheduling Preferences
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbar-dropdown">
+              <a>Working Hours:</a>
+              <form class="px-4 py-3">
+                <input
+                  type="time"
+                  id="working-hour-start"
+                  name="working-hour-start"
+                  class="form-control"
+                  value="09:00"
+                />
+                <a>to</a>
```

Comment:
Does this need to be a tag? Can you use a span tag instead?

Reply:
Done.